pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/ModuleVersionUpload.java

```diff
@@ -90,6 +90,7 @@ protected void doRun(SlingHttpServletRequest request, PostResponse response, Lis
         try {
             String locale = ServletUtils.paramValue(request, "locale", GlobalConfig.DEFAULT_MODULE_LOCALE.toString());
             String asciidocContent = ServletUtils.paramValue(request, "asciidoc");
+            asciidocContent = new String(asciidocContent.getBytes(request.getCharacterEncoding()), "utf-8");
```

Comment:
Would you prefer to use something like `StandardCharsets.UTF_8`?

Reply:
Fixed